database/index.js

```diff
@@ -66,8 +66,16 @@ User.findById(data.project_id, function (err, user) {
 			  project.save();
            });
 }
-var deleteProject = function(data,callback){
-	//console.log("Dataaaa",data)
+var deleteProject = function(data,userId,callback){
+	User.findById(userId,function(err,user){
+		if(err){throw err}
+			for(var i=0; i<user.projects.length;i++){
```

Comment:
Don't need to have this indented this far right, since it's not part of the above if statement.